script/ynEigen/input/lsd-mainnet.json

```diff
@@ -19,9 +19,9 @@
       "strategy": "0xa4C637e0F704745D182e4D38cAb7E7485321d059"
     },
     {
-      "name": "RETH",
-      "token": "0xae78736Cd615f374D3085123A210448E74Fc6393",
-      "strategy": "0x1BeE69b7dFFfA4E2d53C2a2Df135C388AD25dCD2"
+      "name": "METH",
```

Comment:
after update by RaidGuild this descriptor file checked for RETH instead of METH - updated.